app/models/katello/smart_proxy_sync_history.rb

```diff
@@ -0,0 +1,8 @@
+module Katello
+  class SmartProxySyncHistory < Katello::Model
+    self.table_name = 'katello_smart_proxy_sync_history'
+
+    belongs_to :smart_proxy, :class_name => "::SmartProxy", :dependent => :destroy, :inverse_of => :smart_proxy_sync_histories
+    belongs_to :repository, :class_name => "Katello::Repository", :dependent => :destroy, :inverse_of => :smart_proxy_sync_histories
```

Comment:
I think you need to remove these :dependent => :destroy blocks.  I manually destroyed the sync history in the rails console and it blew away my repos and smart proxy ;)

Reply:
Wow..definitely don't want that..:open_mouth: